### PR TITLE
feat: prisma accelerate

### PR DESCRIPTION
**Pull request by pondorasti**
- [x] enable edge rendering

### CODE REVIEW

File: src/server/get-emojis-count.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
 import "server-only"
 import { prisma } from "./db"
+import ms from "ms"
 
-export const revalidate = 1_800 // revalidate the data at most every 30 minute
-
-export const getEmojisCount = cache(async () => prisma.emoji.count())
+export const getEmojisCount = cache(async () =>
+  prisma.emoji.count({
+    cacheStrategy: { ttl: ms("5m"), swr: ms("5m") },
```
Comment (ankur-arch) [inline src/server/get-emojis-count.ts]:
```suggestion
    cacheStrategy: { ttl: ms("5m")/1000, swr: ms("5m")/1000 },
```

Comment (ankur-arch) [reply src/server/get-emojis-count.ts]: The time should be added in seconds rather than milliseconds

File: src/server/get-emojis.ts [hunk 2]
```diff
@@ -2,14 +2,14 @@ import { Prisma } from "@prisma/client"
...
-export const revalidate = 60 // revalidate the data at most every 1 minute
+import ms from "ms"
 
 export const getEmojis = cache(async (take: number = 100) =>
   prisma.emoji.findMany({
     select: { id: true, updatedAt: true },
     orderBy: { createdAt: Prisma.SortOrder.desc },
     where: { isFlagged: false, originalUrl: { not: null }, noBackgroundUrl: { not: null } },
     take,
+    cacheStrategy: { ttl: ms("30s"), swr: ms("60s") },
```
Comment (ankur-arch) [inline src/server/get-emojis.ts]:
```suggestion
    cacheStrategy: { ttl: ms("30s")/1000, swr: ms("60s")/1000 },
```


 Same case for here 😄

**Comment by ankur-arch** [comment]
Summary:
- convert all the routes to edge routes
- disable caching explicitly in the route handlers, 
   Prisma Accelerate relies on HTTP to communicate with the database, and Vercel automatically caches all HTTP requests, other than POST requests, on the POST route handler. 
 - the memoization with the cache calls can be skipped because Accelerate's cache is faster with more utilization

**Comment by pondorasti** [comment]
Thanks for the thoughtful summary @ankur-arch. I'm getting this merged with accelerate enabled to test stability in prod, and then I will do a fast follow for the ttl/swr headers.

**Pull request merged**